### PR TITLE
Update data for 3D tiles example

### PR DESCRIPTION
**Pull request by jianhuang01**
#### Change List
- update token and data
- add thumbnail

### CODE REVIEW

File: examples/website/3d-tiles/app.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class App extends PureComponent {
         // Update deck.gl viewState, moving the camera to the new tileset
         longitude: cartographicCenter[0],
         latitude: cartographicCenter[1],
-        zoom: zoom + 1.5, // TODO - remove adjustment when Tileset3D calculates correct zoom
+        zoom,
```
Comment (ibgreen) [inline examples/website/3d-tiles/app.js]: Not too zoomed out?

Comment (xintongxia) [inline examples/website/3d-tiles/app.js]: is it corrected?

Comment (jianhuang01) [reply examples/website/3d-tiles/app.js]: I think it is better to zoom out a little on the website, users can understand the point cloud better. In high zoom, the model is kind of too sparse, people may not notice the full picture.

**Comment by jianhuang01** [comment]
@ibgreen @xintongxia 
image

File: examples/website/3d-tiles/app.js (hunk 1 above)
Comment (jianhuang01) [reply examples/website/3d-tiles/app.js]: Trying to make the model has a higher density, easier to perceive.

**Pull request merged**